po/ca.po

```diff
@@ -1054,7 +1054,7 @@ msgid "Automatically using specific NTP servers"
 msgstr "Automàticament mitjançant servidors NTP específics"
 
 #: pkg/machines/components/vmOverviewTabLibvirt.jsx:70
-msgid "Autostart:"
+msgid "Autostart"
 msgstr "Inici automàtic:"
```

Comment:
Well, *this* part happens automatically with `make update-po` (and then translation would be marked as "fuzzy"). The point was to drop the colon from the msgstr as well, so that there is a current and matching translation. This will require some advanced seddery, let me know if you want help with this.